contracts/dtoken/tests/sim/scenarious.rs

```diff
@@ -424,6 +424,72 @@ fn scenario_supply_not_enough_balance() {
     );
     assert_failure(result, "The account doesn't have enough balance");
 }
+#[test]
+fn triple_supply_success() {

```

Comment:
@dmytro-shash 
code should do only what is needed, your intention was to check that it's possible to call one user action after another so it's redundant to do that once again

You can check that you test has redundant steps if tou write it in Given-When-Then notation and see that you have repeated actions

Given: Account is has not been locked yet
(1st call of supply)
When: Supply action started
Then: Account unlocked
When: Supply action finished
Then: Account unlocked and could perform action
(2nd call of supply)
When: Supply action started
Then: Account unlocked
When: Supply action finished
Then: Account unlocked and could perform action
(3rd call of supply)
When: Supply action started
Then: Account unlocked
When: Supply action finished
Then: Account unlocked and could perform action

As you can see 2nd and 3rd cases has identical states so that means that you test has redundant steps

In general you test not supply but `two_actions_in_a_row` so it's better name.
But to be sure, could you rewrite this test to be `supply_borrow_repay_withdraw` so we sure that user can perform all this actions in a row.